app/src/main/java/com/odysee/app/ui/findcontent/FileViewFragment.java

```diff
@@ -1727,23 +1736,41 @@ private void checkAndLoadComments() {
     private void checkAndLoadComments(boolean forceReload) {
         View root = getView();
         if (root != null) {
+            View expandCommentArea = root.findViewById(R.id.expand_commentarea_button);
             View commentsDisabledText = root.findViewById(R.id.file_view_disabled_comments);
             RecyclerView commentsList = root.findViewById(R.id.file_view_comments_list);
-            if (claim.getTags().contains("disable-comments") || claim.getSigningChannel().getTags().contains("disable-comments")) {
-                root.findViewById(R.id.expand_commentarea_button).setVisibility(View.GONE);
-                Helper.setViewVisibility(commentsDisabledText, View.VISIBLE);
-                Helper.setViewVisibility(commentsList, View.GONE);
-            } else {
-                root.findViewById(R.id.expand_commentarea_button).setVisibility(View.VISIBLE);
-                Helper.setViewVisibility(commentsDisabledText, View.GONE);
-                Helper.setViewVisibility(commentsList, View.VISIBLE);
-                if ((commentsList != null && forceReload) || (commentsList == null || commentsList.getAdapter() == null || commentsList.getAdapter().getItemCount() == 0)) {
-                    loadComments();
+
+            showComments(expandCommentArea, commentsDisabledText, commentsList);
```

Comment:
Same flicker issue as above. Better to hide the display area before doing the disabled check.